### ISSUE TITLE
Fix: MD023 Doc: exceptions

### ISSUE THREAD

**Issue by Malix-off** [DavidAnson/markdownlint#1009]
# Fix

## Subject

[MD023 Documentation](https://github.com/DavidAnson/markdownlint/blob/main/doc/md023.md)

## Object

Should state the exceptions:
- Lists
```md
- ## this header still complies with MD023 
```
- Blockquotes
```md
> # Heading
```

**Comment by Malix-off** [comment]
- [Comment Reply](https://github.com/DavidAnson/markdownlint/issues/1009#issuecomment-1766937861) from @DavidAnson:
  > What exception for lists are you referring to?

  See [this issue's description](https://github.com/DavidAnson/markdownlint/issues/1009#issue-1947957048)  @ Object/Exemple

**Comment by DavidAnson** [comment]
Thanks for the edit. It's the same if they are nested in a blockquote. I can clarify that in the documentation.

**Comment by Malix-off** [comment]
- [Comment Reply](https://github.com/DavidAnson/markdownlint/issues/1009#issuecomment-1766995785) from @DavidAnson:
  > [⋯](https://github.com/DavidAnson/markdownlint/issues/1009#issuecomment-1766995785) It's the same if they are nested in a blockquote [⋯](https://github.com/DavidAnson/markdownlint/issues/1009#issuecomment-1766995785)

  Can you provide an example?

**Comment by DavidAnson** [comment]
Yes, this is not reported as an error:

```Md
> # Heading
```
